src/GitHub.VisualStudio/Views/GitHubPane/PullRequestListItemView.xaml

```diff
@@ -58,33 +58,28 @@
                                    Height="30"
                                    VerticalAlignment="Top"/>
 
-            <ghfvs:OcticonImage DockPanel.Dock="Right"
-                                Background="Transparent"
-                                Icon="git_pull_request"
-                                ToolTip="This is the current branch"
-                                VerticalAlignment="Stretch"
-                                Visibility="{Binding IsCurrent, Converter={ghfvs:BooleanToVisibilityConverter}}"/>
```

Comment:
I removed this to keep the number of icons down in the list, but I still think it's useful to call out if you have checked out a branch with a pull request associated with it.

One idea that @grokys and I kicked around is bolding the title. This is similar to a treatment used in the "Connect" page showing which repository that you are currently in:

<img width="304" alt="screen shot 2018-06-29 at 10 44 30 am" src="https://user-images.githubusercontent.com/1174461/42106596-6fd93c4a-7b89-11e8-9aec-c07bee9ebeff.png">